app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb

```diff
@@ -6,6 +6,12 @@ oses:
 - CentOS
 - Fedora
 - RedHat
+test_on:
+- host
```

Comment:
Thoughts about renaming this to `host4` or even `host4dhcp`?

Reply:
Yeah I was thinking the same, I will do this.